presto-hive/src/test/java/com/facebook/presto/hive/TestHivePushdownFilterQueries.java

```diff
@@ -84,6 +88,32 @@ public void testBooleans()
         assertQueryUsingH2Cte("SELECT COUNT(*) FROM lineitem_ex WHERE ship_by_air is not null AND is_returned = true");
     }
 
+    @Test
+    public void testBytes()
+    {
+        // Single tinyint column
+        assertQueryUsingH2Cte("SELECT byte_col1 FROM lineitem_ex");
+
+        assertQueryUsingH2Cte("SELECT byte_col1 FROM lineitem_ex WHERE byte_col1 < 0");
```

Comment:
This query returns no data. Is this intentional? If so, perhaps, use `assertQueryReturnsEmptyResult`.

Reply:
using ship_day and ship_month as suggested above